<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateImageDialog</class>
 <widget class="QDialog" name="CreateImageDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>259</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create a disk image</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="stdSingleButton">
        <property name="text">
         <string>Standard single density</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="stdEnhancedButton">
        <property name="text">
         <string>Standard enhanced (also called medium or dual) density</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="stdDoubleButton">
        <property name="text">
         <string>Standard double density</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="doubleDoubleButton">
        <property name="text">
         <string>Double sided double density</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="harddiskButton">
        <property name="text">
         <string>Double density harddisk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="customButton">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="geometryWidget" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Number of sectors:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sectorsSpin">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>65535</number>
             </property>
             <property name="value">
              <number>720</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Sector density:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="densityCombo">
             <item>
              <property name="text">
               <string>Single (128 bytes per sector)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Double (256 bytes per sector)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>512 bytes per sector</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8192 bytes per sector</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="capacityLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Total image capacity: 92160 bytes (90 K)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateImageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateImageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
